tests/tools/onert_train/src/nnfw_util.h

```diff
@@ -36,9 +37,16 @@ namespace onert_train
 uint64_t num_elems(const nnfw_tensorinfo *ti);
 uint64_t bufsize_for(const nnfw_tensorinfo *ti);
 
+// TODO: update to use snake case
 std::string toString(NNFW_TRAIN_OPTIMIZER opt);
 std::string toString(NNFW_TRAIN_LOSS loss);
 std::string toString(NNFW_TRAIN_LOSS_REDUCTION loss_rdt);
 std::ostream &operator<<(std::ostream &os, const nnfw_train_info &info);
+
+template <typename E> constexpr auto to_underlying(E e) noexcept
+{
+  return static_cast<std::underlying_type_t<E>>(e);
+}
```

Comment:
(note) to find the underlying value(usually integer) for enum

(I referred this function from modern c++ item 10)

Reply:
Oh! I've never seen this before. Interesting!

Reply:
Naming is `to_underlying`(snake). Above functions' names are `toString` (camel). Which one is right?

Reply:
:+1: :+1: 
resolved by : https://github.com/Samsung/ONE/pull/12577